Remove hardcode password for first user and the root passwords in Agama unattended jsonnet profiles

lemon-suse opened the pull request:
- Related ticket: https://progress.opensuse.org/issues/168853
- Needles: N/A
- Verification run: 
  Agama devel group: 
  x86_64:
  https://openqa.opensuse.org/tests/4691164   agama_auto_default
  https://openqa.opensuse.org/tests/4691200#details   agama_auto_gnome
  ppc64le:
 https://openqa.opensuse.org/tests/4691181
aarch64:
https://openqa.opensuse.org/tests/4691180

jknphy commented on data/yam/agama/auto/default_tumbleweed.sh:
```diff
@@ -1,7 +1,7 @@
 set -ex
```
we don't use this, we can drop this file.

lemon-suse replied:
removed. So later we won't add such test for profile file extension of sh?

jknphy replied:
I don't think now it is even an option this kind of scripts, that was an ALP prototype.

jknphy commented on data/yam/agama/auto/leap16.sh:
```diff
@@ -1,7 +1,7 @@
 set -ex
```
we don't use this, we can drop this file.

lemon-suse replied:
removed

jknphy commented:
Looks like you are passing a long non-encrypted password, most likely you need to activate the flag of encrypted password, please check the documentation.

lemon-suse commented:
> Looks like you are passing a long non-encrypted password, most likely you need to activate the flag of encrypted password, please read again the documentation:
> 
> ```
> Similar to AutoYaST, an additional encryptedPassword boolean flag is used to determine whether the specified password is encrypted (true value) or plain text (false value or missing in the profile)
> ```

Thanks, added the flag for encrypted password, updated VRs also.
